docs/lob_guide/src/asciidoc/_chapters/work_with_lob.adoc

```diff
@@ -0,0 +1,738 @@
+```
+
+* This example inserts the c1 and c2 selected from the source Trafodion table t130var into the c1 and c2 of the target table t130lob2.
++
+```
+INSERT INTO t130lob2(c1,c2) SELECT c1,c2 FROM t130var;
+```
+
+* This example inserts the c1 and c2 selected from the source Trafodion table t130char into the c1 and c2 of the target table t130lob2.
++
+```
+INSERT INTO t130lob2(c1,c2) SELECT c1,c2 FROM t130char;
+```
+
+* This example inserts the c1, c2 and c3 selected from the source Trafodion table t130lob2 into the target table t130lob3, and then shows the result.
++
+```
+INSERT INTO t130lob3 SELECT c1,c2,c3 FROM t130lob2;
+SELECT c1,lobtostring(c2,100),lobtostring(c3,100) FROM t130lob3;
+```
+
+[#update a sql table containing lob columns]
+== Update a SQL Table Containing LOB Columns
+
+The source for updating a LOB can be divided into 2 groups:
+
+* Update using parameters/functions
+* Update using LOB handle
+
+[#update using parameters/functions]
+=== Update Using Parameters/Functions
+
+The following parameters/functions can be used to update.
+
+* A parameter.
+* `EMPTY_BLOB` or `EMPTY_CLOB`.
+* An in-memory lob which is a simple string data.  
++
+To insert this string, a literal needs to be provided.
+* An on-platform file (linux/hdfs file) containing text or binary data.  
+* A user buffer of a specified length allocated in user space.
+* An external LOB file in HDFS.
+
+[#syntax]
+==== Syntax 
+
+```
+UPDATE table-name {set-clause-type1 | set-clause-type2} 
+```
+
+```
+set-clause-type1 is: 
+SET set-clause[, set-clause ].. 
+
+set-clause is: 	
+lob_column-name = {lob_query-expr} 
+
+lob_query-expr is:
+NULL | ?                                                                                |
+EMPTY_BLOB()                                                                            |
+EMPTY_CLOB()                                                                            |                                                
+STRINGTOLOB('string literal expression'[, APPEND])                                      |
+FILETOLOB('lob source file name'[, APPEND])                                             |
+BUFFERTOLOB(LOCATION lob source buffer address, LENGTH lob length value[, APPEND])      |
+EXTERNALTOLOB('external lob source file name'[, APPEND])
+
+lob source file name is:
+hdfs:///{local hdfs file name}      |  
+{local linux file name}             |
+{file:///linux file name} 
+
+external lob source file name is: 
+hdfs:///{local hdfs file name}
+```
+
+[#semantics]
+==== Semantics
+
+For more information, see Semantics in <<insert into a sql table containing lob columns,Insert into a SQL Table Containing LOB Columns>>.
+
+[#examples]
+==== Examples
+
+* In the table tlob1where c1 is 3, updates (appends) the value of c2 to lob_update.txt.
++
+```
+UPDATE tlob1 SET c2=filetolob('lob_update.txt', append) WHERE c1 = 3;
+```
+
+* In the table tlob1, updates (overwrites) the c2 to anoush.jpg.

```

Comment:
Instead of "the c2", I suggest "the value of c2"

Reply:
Thanks Dave, I've corrected. :)